bundles/org.openhab.binding.panasonicbdp/src/main/resources/OH-INF/thing/uhdplayer.xml

```diff
@@ -119,17 +119,6 @@
 		<autoUpdatePolicy>veto</autoUpdatePolicy>
 	</channel-type>
 
-	<channel-type id="control">
-		<item-type>Player</item-type>
-		<label>Control</label>
-		<description>Transport Controls e.g. Play/Pause/Next/Previous/FForward/Rewind</description>
-		<category>Player</category>
-		<tags>
-			<tag>Control</tag>
-			<tag>MediaControl</tag>
-		</tags>
-	</channel-type>
-
```

Comment:
Why is this removed? Ditto below.

Reply:
Because it is duplicate, as it is defined on both files.